parlai/core/dict.py

```diff
@@ -694,21 +697,6 @@ def sort(self, trim=True):
         assert len(self.freq) == len(self.ind2tok) == len(self.tok2ind)
         return sorted_pairs
 
-    def parse(self, txt_or_vec, vec_type=list):
```

Comment:
This is an unrelated change, but just killing a bad method.

Reply:
Nit: Probably best to put this in a separate change for rubber stamping?